add how to set delimiter (#153)
keboola/developers-docs: integrate/variables/index.md

}
{% endhighlight %}

## Variable Delimiter
The default variable delimiter is `{{ "{{" }}` and `}}`. If the delimiter interferes with your code, it can
be changed as per the [Moustache docs](https://mustache.github.io/mustache.5.html). For example the 
following piece of code

{% highlight json %}
{
    "code": "SELECT \"COUNTRY\" || '{{ "{{ alias " }}}}' || '{{ "{{=<< >>=" }}}} {{ "{{ as-is " }}}} <<={{ "{{ " }}}}=>>' 
    AS \"COUNTRY\", \"CARS\" || '{{ "{{ size " }}}}' AS \"CARS\" FROM \"my-table\""
}
{% endhighlight %}

> Will be interpreted as (assuming that there are variables `alias=batman` and `size=big` defined):

Reviewer: ```suggestion
will be interpreted as (assuming the variables `alias=batman` and `size=big` are defined):
```
